personal_profile/tests.py

```diff
@@ -31,13 +33,38 @@ def profile_1(db, user_1):
     return profile_1
 
 
+@pytest.mark.django_db
 class TestProfileModel:
-    # Testing different Personal Profile attributes
+    # Testing different Personal Profile saving and deletion
 
-    def test_newprofile(self, profile_1, user_1):
+    def test_profile_is_saved_correctly(self, profile_1, user_1):
```

Comment:
If test verifies the profile is saved,
Is should search the profile, and verify its data against fixture content,
same for other tests

Reply:
done